SPECS/orangefs/orangefs.spec

```diff
@@ -86,13 +109,7 @@ autoreconf -vif -I maint/config
 
 %build
 export LDFLAGS="%{optflags} -Wl,--as-needed"
-%ifarch armv7hl
-%configure --enable-external-lmdb --enable-shared --disable-static \
-    --enable-fuse --disable-usrint --with-db-backend=lmdb
-%else
-%configure --enable-external-lmdb --enable-shared --disable-static \
-   --enable-fuse --disable-usrint --with-db-backend=lmdb --with-openib=%{_prefix}
-%endif
+%configure --enable-external-lmdb --enable-shared --disable-static --enable-fuse --disable-usrint --with-db-backend=lmdb --with-openib=%{_prefix}
```

Comment:
Thanks for the update. Could you also refactor to use one switch per line? This way it's easier to track config changes across commits. It's also going to make reading the config easier for everyone.:)

Reply:
updated